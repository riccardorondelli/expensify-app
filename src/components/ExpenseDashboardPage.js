import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => (
  <div>
<<<<<<< HEAD
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
=======
    <ExpenseListFilters />
    <ExpenseList />
    <ExpensesSummary />
>>>>>>> b654511824d8d542aed674707059cbd03c3dcb09
  </div>
);

export default ExpenseDashboardPage;
