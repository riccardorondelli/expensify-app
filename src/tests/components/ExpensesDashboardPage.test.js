import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('should render ExpenseList and ExpenseListFilters', () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});