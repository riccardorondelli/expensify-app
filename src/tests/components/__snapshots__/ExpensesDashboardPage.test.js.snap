// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList and ExpenseListFilters 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
